include/SFML/Window/Keyboard.hpp

```diff
@@ -153,6 +158,158 @@ class SFML_WINDOW_API Keyboard
+        ScanDown,              ///< Keyboard Down Arrow key
+        ScanUp,                ///< Keyboard Up Arrow key
+        ScanNumLock,           ///< Keypad Num Lock and Clear key
+        ScanDivide,            ///< Keypad / key
+        ScanMultiply,          ///< Keypad * key
+        ScanMinus,             ///< Keypad - key
+        ScanPlus,              ///< Keypad + key
+        ScanPadEquals,         ///< keypad = key, probably Mac only
+        ScanReturn,            ///< Keypad Enter (return) key
+        ScanDecimal,           ///< Keypad . and Delete key
+        ScanNumpad1,           ///< Keypad 1 and End key
+        ScanNumpad2,           ///< Keypad 2 and Down Arrow key
+        ScanNumpad3,           ///< Keypad 3 and Page Down key
+        ScanNumpad4,           ///< Keypad 4 and Left Arrow key
+        ScanNumpad5,           ///< Keypad 5 key
+        ScanNumpad6,           ///< Keypad 6 and Right Arrow key
+        ScanNumpad7,           ///< Keypad 7 and Home key
+        ScanNumpad8,           ///< Keypad 8 and Up Arrow key
+        ScanNumpad9,           ///< Keypad 9 and Page Up key
+        ScanNumpad0,           ///< Keypad 0 and Insert key
+        ScanReverseSolidus,    ///< Keyboard Non-US \ and | key
+        // FIXME what is this one? Might need better name. The doc says:
+        // - Typically near the Left-Shift key in AT-102 implementations.
+        // - Typical language mappings: Belg:<\> FrCa:«°» Dan:<\> Dutch:]|[ Fren:<> Ger:<|> Ital:<> LatAm:<> Nor:<> Span:<> Swed:<|> Swiss:<\> UK:\| Brazil: \|.
+        // What is the difference with "regular" \ and | key?
+        ScanApplication,       ///< Keyboard Application key
+        ScanExecute,           ///< Keyboard Execute key
+        ScanHelp,              ///< Keyboard Help key
+        ScanMenu,              ///< Keyboard Menu key
+        ScanSelect,            ///< Keyboard Select key
+        ScanStop,              ///< Keyboard Stop key
+        ScanAgain,             ///< Keyboard Again key
+        ScanUndo,              ///< Keyboard Undo key
+        ScanCut,               ///< Keyboard Cut key
+        ScanCopy,              ///< Keyboard Copy key
+        ScanPaste,             ///< Keyboard Paste key
+        ScanFind,              ///< Keyboard Find key
+        ScanMute,              ///< Keyboard Mute key
+        ScanVolumeUp,          ///< Keyboard Volume Up key
+        ScanVolumeDown,        ///< Keyboard Volume Down key
+        ScanLControl,          ///< Keyboard Left Control key
+        ScanLShift,            ///< Keyboard Left Shift key
+        ScanLAlt,              ///< Keyboard Left Alt key
+        ScanLSystem,           ///< Keyboard Left System key
+        ScanRControl,          ///< Keyboard Right Control key
+        ScanRShift,            ///< Keyboard Right Shift key
+        ScanRAlt,              ///< Keyboard Right Alt key
+        ScanRSystem,           ///< Keyboard Right System key
+
```

Comment:
I'd suggest checking this out: https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h

It has a bunch of additional scancodes like "Calculator" or "Mail".
Some of them may not be ever needed for SFML. Let's discuss it on the forum, maybe?

Reply:
Those other codes are not part of the Keyboard/Keypad Page (0x07) but of the Consumer Page (0x0C). We can discuss adding them later on if they are requested. :)

Reply:
Fair enough. They're not important for now, I think.